spec/descriptions/tagSyntheticResults.md

```diff
@@ -265,6 +265,81 @@ To narrow down the result set you have three options to search for a test.
 }
 ```
 
+## Get a list of Synthetic locations with Last Test Run on (each location) data
+The endpoint returns a list of Synthetic locations with Last Test Run on (each location) result data
+
+### Mandatory Parameters
+
+**timeFrame** As in our UI you can specify the timeframe for metrics retrieval.
+```
+  windowSize           to
+     (ms)       (unix-timestamp)
+<----------------------|
+
+"timeFrame": {
+	"windowSize": 60000,
+	"to": {current timestamp}
+}
+```
+
+### Optional Parameters
+
+**pagination** if you use pagination you most probably want to fix the timeFrame for the retrieved metrics
```

Comment:
Do you mean that if they use pagination, they should also specify a timeframe?  If so then may be it could say "if you use pagination, you should provide the timeframe parameter".  I am not sure what you mean by "fix the timeFrame", unless you just mean to have a set timeframe?

Reply:
I think "fix the timeFrame" means you have to use the same timeFrame for all the pages you want to query. 
If you change it, the elements of the pagination will probably change too.

Reply:
that is more clear to me